docs/zh/react/reactlynx-testing-library/index.mdx

```diff
@@ -30,83 +28,11 @@ const config = defineConfig({
 export default mergeConfig(defaultConfig, config);
 ```
 
-如果需要使用 `@testing-library/jest-dom` 中的 `toBeInTheDocument` 等方法，你需要安装 `@testing-library/jest-dom`：
-
-<PackageManagerTabs command="add -D @testing-library/jest-dom" />
-
-然后你就可以开始编写测试并运行了，下面是一个完整的示例：
-
-```tsx title=src/__tests__/index.test.tsx
-import '@testing-library/jest-dom';
-import { expect, test, vi } from 'vitest';
-import { render, getQueriesForElement } from '@lynx-js/react/testing-library';
-import { Component } from '@lynx-js/react';
-
-export interface IProps {
-  onMounted?: () => void;
-}
-
-export class App extends Component<IProps> {
-  override componentDidMount() {
-    this.props?.onMounted?.();
-  }
-
-  override render() {
-    return (
-      <view>
-        <text id="app-text">Hello World!</text>
-      </view>
-    );
-  }
-}
-
-test('App', async () => {
-  const cb = vi.fn();
-
-  render(
-    <App
-      onMounted={() => {
-        cb(`__MAIN_THREAD__: ${__MAIN_THREAD__}`);
-      }}
-    />,
-  );
-  expect(cb).toBeCalledTimes(1);
-  expect(cb.mock.calls).toMatchInlineSnapshot(`
-    [
-      [
-        "__MAIN_THREAD__: false",
-      ],
-    ]
-  `);
-  expect(elementTree.root).toMatchInlineSnapshot(`
-    <page>
-      <view>
-        <text
-          id="app-text"
-        >
-          Hello World!
-        </text>
-      </view>
-    </page>
-  `);
-  const { findByText } = getQueriesForElement(elementTree.root!);
-  const element = await findByText('Hello World!');
-  expect(element).toBeInTheDocument();
-  expect(element).toMatchInlineSnapshot(`
-    <text
-      id="app-text"
-    >
-      Hello World!
-    </text>
-  `);
-});
-```
-
 ## 示例
 
 ### 快速开始
 
-这是一个最小化的示例，展示了如何使用 ReactLynx Testing Library 进行测试。
+和 React Testing Library 一样，我们推荐将测试用例分为 Arrange、Act 和 Assert 三个部分。Arrange 部分用于准备测试数据，Act 部分用于执行测试操作，Assert 部分用于断言测试结果。下面是一个简单的示例：
```

Comment:
Please translate those words and link them to the original doc. They are natural languages rather than formal definitions or code